### PR TITLE
feat(issues): Implement organization issue detail view

### PR DESCRIPTION
**Pull request by lynnagara**
Depends on:
- [x] https://github.com/getsentry/sentry/pull/11508
- [x] https://github.com/getsentry/sentry/pull/11513
- [x] https://github.com/getsentry/sentry/pull/11539

### CODE REVIEW

File: src/sentry/web/urls.py [hunk 1]
```diff
@@ -399,6 +399,12 @@ def init_all_applications():
     url(r'^(?P<organization_slug>[\w_-]+)/$',
         react_page_view, name='sentry-organization-home'),
     url(r'^organizations/new/$', generic_react_page_view),
+
+    url(
+        r'^organizations/(?P<organization_slug>[\w_-]+)/issues/(?P<issue_id>\d+)/$',
```
Comment (markstory) [inline src/sentry/web/urls.py]: For my own curiosity/learning, why is this needed?

**Review by markstory — APPROVED** [review]

File: src/sentry/web/urls.py (hunk 1 above)
Comment (lynnagara) [reply src/sentry/web/urls.py]: I am honestly not sure why this path had to be explicitly added and wasn't resolving to a react page by default. I couldn't find any other routes that seemed to match this path in this file
